src/Microsoft.ML.Featurizers/Common.cs

```diff
@@ -0,0 +1,189 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using System.Security;
+using System.Text;
+using Microsoft.Win32.SafeHandles;
+
+namespace Microsoft.ML.Featurizers
+{
+    #region Native Function Declarations
+
+    #endregion
+
+    internal enum FitResult : byte
+    {
+        Complete = 1, Continue, ResetAndContinue
+    }
+
+    // Not all these types are currently supported. This is so the ordering will allign with the native code.
+    internal enum TypeId : uint
+    {
+        String = 1, SByte, Short, Int, Long, Byte, UShort,
```

Comment:
Typical C# code has these be on separate lines.

Reply:
Should we have a int next to each? This will assist with long term compatibility by allowing for inserting new values, and in case someone decides to sort the enum differently later (more important if serialized in a model).

Reply:
Yeah, you are correct. I have added an int next to each one now.

Reply:
Thanks